src/sentry/integrations/jira/integration.py

```diff
@@ -394,13 +396,19 @@ def get_create_issue_config(self, group, **kwargs):
 
         # TODO(jess): are we going to allow ignored fields?
         # ignored_fields = (self.get_option('ignored_fields', group.project) or '').split(',')
-        ignored_fields = set()
+
+        # hide sprint, epic link, parent and linked issues fields because they don't work
+        # since sprint and epic link are "custom" we need to search fo rthem by name
+        ignored_fields = set(
+            k for k, v in six.iteritems(issue_type_meta['fields']) if v['name'] in {'Sprint', 'Epic Link'}
+        )
+        ignored_fields.update(['parent', 'issuelinks'])
```

Comment:
Can you document why we have to do this differently for the different files (sprint and epic vs parent and issue link)

Reply:
i did: https://github.com/getsentry/sentry/pull/9421/files#diff-f78e256a535cb9c8f27ccbaa1f1f4ac8R401

Reply:
I can't read lol